### DISCUSSION TITLE
oklch css4 colors

### DISCUSSION THREAD

**Discussion by stafyniaksacha** [tailwindlabs/tailwindcss#10828] [General]
Hello there!

Just wondering if any discussion about implanting `oklch` colors on tailwind have been initiated yet?
The new gamut is really impressive (when hardware and browser support them)

I would love to see a `tailwind/colors-oklch` module that would use p3 colors!

What do you think?

Some readings: 
- https://www.w3.org/TR/css-color-4/
- https://bottosson.github.io/posts/oklab/
- https://evilmartians.com/chronicles/oklch-in-css-why-quit-rgb-hsl

**Comment by rihardsgrislis** [comment]
Just stumbled upon this too, some more reading: 
https://www.kirupa.chat/p/oklch-moving-past-rgb-for-defining
https://oklch.com/#65,0.202,245.65,100
